### PR TITLE
Load coretemp module on boot

### PR DESCRIPTION
**Pull request by MiLk**
- CentOS 6: https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/6/html/Deployment_Guide/sec-Persistent_Module_Loading.html
- CentOS 7: https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/System_Administrators_Guide/sec-Persistent_Module_Loading.html

### CODE REVIEW

**Comment by MiLk** [comment]
cc @tokyowizard

File: tasks/centos.yml [hunk 1]
```diff
@@ -47,3 +47,29 @@
...
     - pkgs
+
+- name: "Load coretemp module on boot"
+  copy: >
+    src=coretemp.modules
+    dest=/etc/sysconfig/modules/coretemp.modules
+    owner=root
+    group=root
+    mode=0700
+    state=present
```
Comment (tokyowizard) [inline tasks/centos.yml]:
```
[14:31:07] TASK: [aeriscloud.collectd | Load coretemp module on boot] ********************
[14:31:09] failed: [collector1] => {"checksum": "53d47070cc9ff9d98e924e8ac0212f3870f88678", "failed": true}
[14:31:09] msg: unsupported parameter for module: state
[14:31:09] !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
[14:31:09] An error has occured while executing this task. See below for more information
[14:31:09] !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
[14:31:09] :::: |invocation::copy|
[14:31:09]       src = coretemp.modules
[14:31:09]       dest = /etc/sysconfig/modules/coretemp.modules
[14:31:09]       owner = root
[14:31:09]       group = root
[14:31:09]       mode = 0700
[14:31:09]       state = present
[14:31:09]
[14:31:09] :::: |msg|
[14:31:09] unsupported parameter for module: state
```

Comment (MiLk) [inline tasks/centos.yml]: updated

**Comment by tokyowizard** [comment]
:+1:

**Pull request merged**